bazel/external/proxy_wasm_cpp_sdk.patch

```diff
@@ -0,0 +1,33 @@
+# Address g++ -std=c++20 variadic template compilation issue:
```

Comment:
I'm against patching this here instead of applying the fix directly to C++ SDK (which we also control) and bumping it here.